[docs] Formatting and proper noun tune-up (#4200)
drud/ddev: docs/content/users/quickstart.md

# CMS Quickstarts

> Once DDEV’s installed you can quickly spin up new projects:

Reviewer: ```suggestion
Once DDEV is installed, you can quickly spin up new projects:
```

6. Run `ddev launch` to open your project in a browser, or visit the URL given by `ddev start`.

!!!tip
    While you’re getting your bearings, use `ddev describe` to get project details, and `ddev help` to investigate commands.

DDEV comes ready to work with any PHP project, and has deeper support for several common PHP platforms and content management systems.

=== "Generic"

    ## Generic

    The `php` project type is the most general, ready for whatever modern PHP or static HTML/JS project you might be working on. It’s just as full-featured as more specific options, just without any app-specific configuration or presets.

    You may even prefer to stick with this flavor despite using one of the apps DDEV supports, simply because you’d rather configure things to your own liking. Please do!
    
    1. Create a directory (`mkdir my-new-project`) or clone your project (`git clone <your_project>`).
    2. Change to the new directory (`cd my-new-project`).
    3. Run `ddev config` and set the project type and docroot, which are usually auto-detected, but may not be if there's no code in there yet.
    4. Run `ddev start`.

>     6. If you’re using Composer build, run `ddev composer install`.

Reviewer: ```suggestion
    6. If you’re using Composer, run `ddev composer install`.
```

    === "Drupal 10"

        ### Drupal 10 via Composer
    

>         [Drupal 10](https://www.drupal.org/about/10) isn’t released yet, but many want to test and contribute to it.

Reviewer: ```suggestion
        [Drupal 10](https://www.drupal.org/about/10) is fully supported by DDEV.
```

        ddev drush uli
        ddev launch
        ```

        !!!tip

>             As Drupal 10 moves from alpha to beta and then release, you’ll want to change the tag from `^10@alpha` to `^10@beta` and then `^10` respectively.

Reviewer: Please change the line above to 
`        ddev composer create --no-install drupal/recommended-project:^10@beta`

```suggestion
            As Drupal 10 moves from beta and to release, you’ll want to change the tag from `^10@beta` to `^10`.
```

ssh-auth status: healthy
```

## Removing Projects

There are two ways to remove a project from DDEV’s listing.

The first is destructive. It removes the project from DDEV’s list, deletes its database, and removes the hostname entry from the hosts file:

`ddev delete <projectname>`  
or  
`ddev delete --omit-snapshot <projectname>`

> The second way only removes the project from `ddev list` without deleting anything else.

Reviewer: ```suggestion
if you just don't want the project to show up in `ddev list` any more, use this nondestructive command to unlist it until the next time you `ddev start` or `ddev config` the project:

```bash
ddev stop --unlist <projectname>
```

> The second way only removes the project from `ddev list` without deleting anything else.

Author: ```suggestion
If you simply don’t want the project to show up in `ddev list` anymore, use this nondestructive command to unlist it until the next time you run `ddev start` or `ddev config`:

```bash
ddev stop --unlist <projectname>
```
```

> Use `ddev stop --unlist <projectname>` to unlist it until the next time you `ddev start` or `ddev config` the project.

Reviewer: ```suggestion
```

Successfully imported database for drupal8
```

#### Supported File Types

Database imports can be any of the following file types:

* Raw SQL Dump (.sql)

> * Gzipped SQL Dump (.sql.gz)

Reviewer: ```suggestion
* Gzipped SQL Dump (`.sql.gz`)
* Xz'd SQL Dump (`.sql.xz`)
```

Author: Bah, good catch! Will try and remember to add backticks for the other file extensions in this list if I missed those too.

Reviewer: `Xz'd` will likely not make the cut with the spellchecker (or you) :)

Author: I’ll admit it’s a verb I’ve never used.
